src/content/en/2021/page-weight.md

```diff
@@ -1,21 +1,176 @@
 ---
 #See https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Authors'-Guide#metadata-to-add-at-the-top-of-your-chapters
 title: Page Weight
-description: TODO
+description: Page Weight chapter of the 2021 Web Almanac covering why page weight matters, bandwidth, complex pages, page weight over time, page requests, and file formats.
 authors: [logicalphase]
 reviewers: [siakaramalegos]
 analysts: [jessthebp]
 editors: [RMHolmlund]
 translators: []
 results: https://docs.google.com/spreadsheets/d/1XApPAC8m2jLJR3Ok1yAoc9PzAFtq-Z41LoakXHUVeag/
-featured_quote: TODO
-featured_stat_1: TODO
-featured_stat_label_1: TODO
-featured_stat_2: TODO
-featured_stat_label_2: TODO
-featured_stat_3: TODO
-featured_stat_label_3: TODO
-
+logicalphase_bio: John currently works as a [Google Cloud Platform](https://cloud.google.com) senior developer and architect. He started his technology journey as a web developer focused on web performance and leveraging browser standards. He applied those principles as a freelance [WordPress](https://wordpress.org) developer, and as an architect and engineer for several managed hosting providers. He is a firm believer in open web standards and sustainable web best practices. To that end, John has worked on several open source projects, including Google's [Lit](https://lit.dev/) project, and is a strong advocate for emerging web technologies such as [Web Components](https://developer.mozilla.org/en-US/docs/Web/Web_Components) and other performance based solutions.
+featured_quote: It's critically important to stop optimizing web pages for &dollar;1,200.00 smartphones with ultra-fast internet connections because the vast majority of internet users don't live in that world, and the energy required to power that world is unsustainable.
+featured_stat_1: 1,923
+featured_stat_label_1: The median number of mobile page bytes
+featured_stat_2: 877
+featured_stat_label_2: The median number of mobile page image bytes
+featured_stat_3: 69
+featured_stat_label_3: The median number of mobile page requests
+unedited: true
 ---
 
-## TODO
+## Introduction
```

Comment:
This is a personal preference, but I kind of like getting a preview of the major findings in the intro.